client/src/App.js

```diff
@@ -1,11 +1,57 @@
+import React,{ useEffect, useState } from "react";
 import "./App.css";
+import VideoCards   from "./VideoCards";
+import AddVideo  from "./AddVideo";
+import data from './exampleresponse.json';
+import OrderButton from "./OrderButton";
+import Footer from "./Footer";
 
 function App() {
+  
+  const [videos, setVideos] = useState(data);
+  const [error, setError] = useState(null);
+  const [order, setOrder] = useState("");
+
+  console.log(data)
+  console.log("videos....")
+  console.log(videos)
+
+useEffect(() => {
+  fetch(`http://localhost:5000/?order=${order}`, {
+    mode: "cors",
+  })
+  .then((res) => {
+    if(!res.ok) throw new Error(`HTTP error! status code ${res.status}`);
+    return res.json();
+  })
+  .then((data) => {
+    const sortedVideos = data.sort((a, b) => b.rating - a.rating);
```

Comment:
You've sent a sort order to the server as a query string but you're re-sorting by rating here